desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -1733,6 +1733,8 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
       self.sampleCols.subscribe(refreshTemporaryTable);
       % if ENABLE_DIRECT_UPLOAD.get():
         self.inputFormat = ko.observable(wizard.prefill.source_type() ? wizard.prefill.source_type() : 'localfile');
+        wizard.prefill.target_type('table');
+        wizard.prefill.source_type('all');
```

Comment:
Thanks for the details.

So we have a 
`|| condition`

https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L206

So not needed if we only have one.

Maybe the problem is that we should also hide if `createWizard.source.inputFormats has only one element`?

Reply:
okay, so I am putting those conditions.